src/encryption/ec.ts

```diff
@@ -3,6 +3,7 @@ import { ec as EllipticCurve } from 'elliptic'
 // @ts-ignore
 import * as BN from 'bn.js'
 import * as crypto from 'crypto'
+import { FailedDecryptionError } from './errors'
```

Comment:
Looks like this is failing tests. I think it needs to be 
```ts
import { FailedDecryptionError } from '../errors'
```

Reply:
Oops, fixed 👍